projects/project-player-library/src/lib/pages/project-details/project-details.component.css

```diff
@@ -13,4 +12,57 @@ box-shadow: rgba(133, 119, 119, 0.555) 5px 5px 4px;
 .project-details p span{
     color: grey;
     font-weight: bold;
+}
+
+.example-headers-align .mat-expansion-panel-header-description {
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .example-headers-align .mat-mdc-form-field + .mat-mdc-form-field {
+    margin-left: 8px;
+  }
+
```

Comment:
All of these style changes are compulsorily required?

Reply:
yes